kda/scripts/verification.py

```diff
@@ -429,12 +429,20 @@ def run_verification(
         # perform various checks and acquire remaining data for G
         check_cycles(G=G, unique_cycles=unique_cycles)
         par_diag_count, dir_par_diag_count = check_diagram_counts(
-            G=G, K=K, dirpars=dirpars, n_states=n_states,
+            G=G,
+            K=K,
+            dirpars=dirpars,
+            n_states=n_states,
```

Comment:
Unnecessary changes. Probably related to `black` operating in my text editor. It doesn't always reformat things, but I should have ran `black` in `kda/scripts` beforehand.